src/widget/sidebar.c

```diff
@@ -1,4 +1,5 @@
 #include "sidebar.h"
+#include "extra_info.h"
```

Comment:
For any includes which are not the corresponding header: please put them in alphabetical order in the list below, including directory prefix. Reasoning: if you move this file, all includes will still work OK.